.github/workflows/auto-update-operators.yml

```
@@ -0,0 +1,24 @@
+name: Auto Update Kubebuilder Operators
+
+on:
+  schedule:
+    - cron: "0 0 * * 2" # Every Tuesday at 00:00 UTC
+  workflow_dispatch:
+
+permissions:
+  contents: write
+  issues: write
+  models: read
```

actionlint (1.7.7)

unknown permission scope "models". all available permission scopes are "actions", "attestations", "checks", "contents", "deployments", "discussions", "id-token", "issues", "packages", "pages", "pull-requests", "repository-projects", "security-events", "statuses"

(permissions)